website/src/components/Visualisations/IIGO/IIGO.tsx

```diff
@@ -1,15 +1,124 @@
+                                        <ListItemText key = {variable} inset = {true} secondary={variable} />
+                                    ))}
+                                </List>
+                            </Typography>
+                            <Typography variant="h6" gutterBottom component="div">
+                                History
+                            </Typography>
+                            <Table size="small" aria-label="purchases">
+                                <TableHead>
+                                    <TableRow>
+                                        <TableCell>Season</TableCell>
+                                        <TableCell align="right">Turn</TableCell>
+                                    </TableRow>
+                                </TableHead>
+                                <TableBody>
+                                    {row.history.map((historyRow) => (
+                                        <TableRow key={historyRow.season*365 + historyRow.turn}>
+                                            <TableCell component="th" scope="row">
+                                                {historyRow.season}
+                                            </TableCell>
+                                            <TableCell align="right">{historyRow.turn}</TableCell>
+                                        </TableRow>
+                                    ))}
+                                </TableBody>
+                            </Table>
+                        </Box>
+                    </Collapse>
+                </TableCell>
+            </TableRow>
+        </React.Fragment>
+    );
+}
+
 
 const IIGO = (props: { output: OutputJSONType }) => {
-  return (
-    <div className={styles.root}>
-      <img src={logo} className={styles.appLogo} alt="logo" />
-      <p className={styles.text}>IIGO Visualisation</p>
-    </div>
-  )
+
+    const rows:RuleType[] = processRulesData(props.output);
+
+    return (
+        <div className={styles.root}>
+            <p className={styles.text}>Rules</p>
+            <TableContainer style={{maxHeight: 500, overflow: 'auto'}} component={Paper}>
+                <Table aria-label="collapsible table">
+                    <TableHead>
+                        <TableRow>
+                            <TableCell />
+                            <TableCell>Rule Name</TableCell>
+                            <TableCell align="right">Mutable</TableCell>
+                            <TableCell align="right">Linked</TableCell>
+                        </TableRow>
+                    </TableHead>
+                    <TableBody>
+                        {rows.map((row) => (
+                            <Row key={row.ruleName} row={row} />
+                        ))}
+                    </TableBody>
+                </Table>
+            </TableContainer>
+        </div>
+  );
 }
 
-export default IIGO
+
```

Comment:
You could remove these extra line spaces but feel free to ignore this comment is pedantic